packages/react-client/src/ReactFlightReplyClient.js

```diff
@@ -187,9 +187,17 @@ export function processReply(
 
   function serializeTypedArray(
     tag: string,
-    typedArray: ArrayBuffer | $ArrayBufferView,
+    typedArray: $ArrayBufferView,
   ): string {
-    const blob = new Blob([typedArray]);
+    const blob = new Blob([
+      // We should be able to pass the buffer straight through but Node < 18 treat
+      // multi-byte array blobs differently so we first convert it to single-byte.
```

Comment:
These are only relevant in Server-to-Server calls and maybe SSR serialization of forms. The question is do we support old Node without polyfilling?

Reply:
IMO we should only support supported versions of Node so we can probably assume FormData